libmamba/src/core/query.cpp

```diff
@@ -156,26 +157,44 @@ namespace mamba
 
     namespace
     {
-        auto print_solvable(const PackageInfo& pkg)
+        auto print_solvable(const PackageInfo& pkg, const std::vector<PackageInfo>& otherBuilds)
         {
+            std::map<std::string, std::vector<PackageInfo>> buildsByVersion;
```

Comment:
Consider using `unordered_map` here, although not sure if it will fit the usage.

Reply:
We want to maintain the order of versions, so the map needs to be ordered.

Reply:
Yes, we need to maintain the order here, hence the `map`.